charts/argo-workflows/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v3.4.4
 name: argo-workflows
 description: A Helm chart for Argo Workflows
 type: application
-version: 0.22.4
+version: 0.22.5
 icon: https://raw.githubusercontent.com/argoproj/argo-workflows/master/docs/assets/argo.png
 home: https://github.com/argoproj/argo-helm
 sources:
```

Comment:
Hi @huiwq1990 thank you for your contribution!
Can you please update (delete old one and add new one) `artifacthub.io/changes`  at Chart.yaml, like below? 🙋 

ref: https://github.com/argoproj/argo-helm/blob/main/charts/argo-workflows/Chart.yaml#L14-L16a

```diff
annotations:
  artifacthub.io/changes: |
-    - "[Added]: Changelog link on README"
+    - "[Added]: Suuport envFrom for main container"
```